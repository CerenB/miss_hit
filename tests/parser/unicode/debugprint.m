function debugprint(s)
    fprintf("string: <%s>\n", s);
    fprintf("length: %u\n", length(s));
end
