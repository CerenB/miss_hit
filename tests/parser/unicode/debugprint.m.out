----------------------------------------------------------------------
--  Parse tree for debugprint.m
Root: Function_File
  Functions: Function_Definition
    Signature: Function_Signature
      Name: Identifier <debugprint>
      Inputs: Identifier <s>
    Body: Sequence_Of_Statements
      Statements: Naked_Expression_Statement
        Expression: Reference
          Name: Identifier <fprintf>
          Arguments: String_Literal "string: <%s>\n"
          Arguments: Identifier <s>
      Statements: Naked_Expression_Statement
        Expression: Reference
          Name: Identifier <fprintf>
          Arguments: String_Literal "length: %u\n"
          Arguments: Reference
            Name: Identifier <length>
            Arguments: Identifier <s>
----------------------------------------------------------------------
In debugprint.m, line 1
| function debugprint(s)
|                    ^ info: this token is not linked to the ast
In debugprint.m, line 1
| function debugprint(s)
|                      ^ info: this token is not linked to the ast
In debugprint.m, line 2
|     fprintf("string: <%s>\n", s);
|            ^ info: this token is not linked to the ast
In debugprint.m, line 2
|     fprintf("string: <%s>\n", s);
|                             ^ info: this token is not linked to the ast
In debugprint.m, line 2
|     fprintf("string: <%s>\n", s);
|                                ^ info: this token is not linked to the ast
In debugprint.m, line 2
|     fprintf("string: <%s>\n", s);
|                                 ^ info: this token is not linked to the ast
In debugprint.m, line 3
|     fprintf("length: %u\n", length(s));
|            ^ info: this token is not linked to the ast
In debugprint.m, line 3
|     fprintf("length: %u\n", length(s));
|                           ^ info: this token is not linked to the ast
In debugprint.m, line 3
|     fprintf("length: %u\n", length(s));
|                                   ^ info: this token is not linked to the ast
In debugprint.m, line 3
|     fprintf("length: %u\n", length(s));
|                                     ^ info: this token is not linked to the ast
In debugprint.m, line 3
|     fprintf("length: %u\n", length(s));
|                                      ^ info: this token is not linked to the ast
In debugprint.m, line 3
|     fprintf("length: %u\n", length(s));
|                                       ^ info: this token is not linked to the ast
In debugprint.m, line 4
| end
| ^^^ info: this token is not linked to the ast
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
